workspaces/copilot/plugins/copilot-common/src/index.ts

```diff
@@ -23,6 +23,7 @@
 /**
  * Represents metrics data for a day in the copilot plugin.
  *
+ * @deprecated See #2828
```

Comment:
I would just removed this, no point in keeping it around given the context of the change, same with the one below.

Reply:
We can't remove this as `convertToMetric()` relies on it (it converts the old API style to the new API style), but I can make it internal for now as it has no use being a public API if it's a breaking change.

New usage should use CopilotMetrics, while internally it can still use Metric.